docs: add 1.6 AGW upgrade docs (#7878)
magma/magma: docs/readmes/lte/deploy_install.md

  You will need two ethernet ports. We use enp1s0 and enp2s0 in this guide.
  They might have different names on your hardware so just replace enp1s0 and
  enp2s0 with your current interfaces name in this guideline.
  One port is for the SGi interface (default: enp1s0) and one for the S1
  interface (default: enp2s0). Note that the `agw_install_ubuntu.sh` script will
  rename the `enp1s0` interface to `eth0`.

## Deployment

### 1. Create boot USB stick and install Ubuntu on your AGW host

- Download Ubuntu Server 20.04 LTS .iso image from Ubuntu website
- Create bootable usb using etcher [tutorial here](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-macos#0)
- Boot your AGW host from USB
  (Press F11 to select boot sequence, :warning: This might be different for
  your machine). If you see 2 options to boot from USB, select the non-UEFI
  option.
- Select “Install” option.
- Network missing firmware "No"
- Primary network interface "enp1s0"
- Configuration.
  * hostname: <choose host name>. Default is "magma". You need to select unique
    host name in case of multiple AGW in network.

>   * Domain name : “”

Reviewer: Nit: here and elsewhere, can you grep-replace all the curly quotes to be straight quotes

---
id: deploy_install
title: Install AGW
hide_title: true
---

> # Install Access Gateway on Ubuntu (Bare Metal)

Reviewer: From previous comment: add a big note here somewhere that Debian installs are deprecated + link to last doc+version with debian support

---
id: deploy_install
title: Install AGW
hide_title: true
---

> > NOTE: AGW deployment on Debian distribution is DEPRECATED! Release 1.5 is last release supporting debian based AGW. Refer to version 1.5 for instructions on [installing AGW on Debian](https://magma.github.io/magma/versions)

Reviewer: ```suggestion
> NOTE: Debian-based AGW deployments are no longer supported as of v1.6. If you want to install to Debian, refer to [v1.5 of the documentation](https://magma.github.io/magma/versions).
```

*Run AGW installation*

> To install on server with DHCP configured SGi interface.
> ```bash

Reviewer: ```suggestion
To install on server with DHCP configured SGi interface.

```bash
```

su

> wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/deploy/agw_install_ubuntu.sh

Reviewer 2: let's do v1.6 instead of master so that remains true when we release 1.7

bash agw_install_ubuntu.sh
```

> To Install on server with statically allocated SGi interface. Fow example:
> SGi has 1.1.1.1/24 IP and upstream router IP is 1.1.1.200
> ```bash

Reviewer: ```suggestion
To Install on server with statically allocated SGi interface. Fow example:
SGi has 1.1.1.1/24 IP and upstream router IP is 1.1.1.200

```bash
```
